### PR TITLE
Fix on scrollbar too big

### LINKED ISSUE #9588: [BOOM-5964] Scroll bar in back-office menu is too big on small screens
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5964).

- _**Reporter:**_ coline.ginestet
- _**Created at:**_ Thu, 12 Jul 2018 16:09:31 +0200

Current scroll bar in 1.7.4.0 is to be changed with this one: https://github.com/mdbootstrap/perfect-scrollbar

image-2018-07-12-16-09-25-564

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Scrollbar was too big so a plugin for scrollbar was asked to be installed. This PR fixes the issue
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #9588 
| How to test?  | Scrollbar on sidebar in BO

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13385)

### CODE REVIEW

File: admin-dev/themes/default/js/vendor/OverlayScrollbars.js [hunk 1]
```diff
@@ -0,0 +1,6222 @@
...
+ *
+ * Version: 1.6.3
+ *
+ * Copyright KingSora.
+ * https://github.com/KingSora
+ *
+ * Released under the MIT license.
+ * Date: 31.01.2019
+ */
+
```
Comment (rdy4ever) [inline admin-dev/themes/default/js/vendor/OverlayScrollbars.js]: I can see that you're inclined into adding heavy jquery plugins for simple tasks. I think you need to understand that performance should always prime over "let's make this little thing a bit prettier by adding a xx+ KB jquery plugin to be loaded on each page". 
If you really want to make a prettier scrollbar, just use [::-webkit-scrollbar](https://css-tricks.com/custom-scrollbars-in-webkit/) to style it for webkit browsers. It has [pretty good browser support](https://caniuse.com/#feat=css-scrollbar).

Comment (YeLnatSs) [reply admin-dev/themes/default/js/vendor/OverlayScrollbars.js]: The plugin has been set up to fix a bug on ie11 and edge.
It was already planned to use 'http://utatti.github.io/perfect-scrollbar/' but I used this one.
"::-webkit-scrollbar" doesn't work for ie11.

Comment (rdy4ever) [reply admin-dev/themes/default/js/vendor/OverlayScrollbars.js]: It was proposed it in #9588, indeed, but [perfect-scrollbar ](https://github.com/utatti/perfect-scrollbar/blob/master/dist/perfect-scrollbar.min.js) weights only 17.9 KB while your solution is much heavier (you added the unminified [330 KB version](https://github.com/PrestaShop/PrestaShop/blob/fbc3ce081a515149d5153b023e56ef788a3b81d5/admin-dev/themes/default/js/vendor/OverlayScrollbars.js)). Styling the scrollbar by css may indeed not work in all browsers, but virtually adds little to no additional weight to the backoffice, which is a win.

Comment (jolelievre) [reply admin-dev/themes/default/js/vendor/OverlayScrollbars.js]: At the very least, if you want to use this plugin, you should add the minified version.

File: admin-dev/themes/default/css/vendor/OverlayScrollbars.min.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/*!
```
Comment (PierreRambaud) [inline admin-dev/themes/default/css/vendor/OverlayScrollbars.min.css]: These files can be removed, you already add it in package.json

Comment (YeLnatSs) [reply admin-dev/themes/default/css/vendor/OverlayScrollbars.min.css]: I was able to install the dependancy and I made the change.
You can check it.

**Comment by PierreRambaud** [comment]
Closed in favor of #14689
Thanks @YeLnatSs

**Pull request closed without merging**